### ISSUE TITLE
Style randomizer: ensure colors are accessible 

### ISSUE THREAD

**Issue by annezazu** [WordPress/gutenberg#47296]
## What problem does this address?

This has come up in the [twentieth call for testing for the FSE Outreach Program](https://make.wordpress.org/test/2023/01/11/fse-program-testing-call-20-find-your-style/#comment-2894):

>However it produces some non-accessible colors for Primary, Secondary and Tertiary, at least over a black background.

While this was previously discussed some here https://github.com/WordPress/gutenberg/pull/40988#issuecomment-1316048575 I wanted to be sure there was a separated out issue for it. 

## What is your proposed solution?

Ensure some level of accessibility friendly calculations/randomizations. cc @vcanales

**Comment by annezazu** [comment]
>Randomize colors had some button color contrast issues. Can we create rules that test for contrast that meets accessibility. I noticed this more quickly when in Storybook button view vs editing home page template.

This came up once more in the [twentieth call for testing for the FSE Outreach Program.](https://make.wordpress.org/test/2023/01/11/fse-program-testing-call-20-find-your-style/#comment-2917)

**Comment by youknowriad** [comment]
Closing as the experiment has been removed. thanks
